src/components/firebase-access.jsx

```diff
@@ -131,11 +130,11 @@ export const initializeQuiz = async (symbols, roomId, periodLen, endDates) => {
   formData.append('RoomId',roomId);
   formData.append('end-date',JSON.stringify(endDates));
   var token =  localStorage.getItem('Token');
-  try{
+  try {
       let response = await fetch("/get_prices", {
         method: 'POST',
         body: formData,
-        //Comment this line back in when you want to deploy , and get rid of localhost
+        //Comment this line back out when you want to run on localhost and add localhost, and comment out headers
         headers: {
           Authorization: ("Bearer " + token)
         },
```

Comment:
like I mentioned before, can you just add a bool at the top of the file `useLocalhost = false` and then modify these HTTP requests to align with whatever the bool value is?

Reply:
That's a more intensive commit, right now. That can be handled later.